spring-cloud-previews/pom.xml

```diff
@@ -9,10 +9,12 @@
   </parent>
   <modelVersion>4.0.0</modelVersion>
 
-  <artifactId>generated-parent</artifactId>
+  <artifactId>spring-cloud-previews</artifactId>
   <packaging>pom</packaging>
 
   <modules>
+    <module>google-cloud-language-spring-starter</module>
+    <module>google-cloud-vision-spring-starter</module>
   </modules>
 
   <build>
```

Comment:
In current setup, build would fail on checkstyle because `maven-checkstyle-plugin` from spring-cloud-gcp/pom runs on `validate` phase but  `fmt-maven-plugin` runs at [a later phase](https://github.com/spotify/fmt-maven-plugin#overriding-the-default-lifecycle-phase). I can think of a few options here:
- @emmileaf based on your work earlier introducing `fmt-maven-plugin`, by any chance we can use only one of the two formatter?
- Adjust running phase so `fmt-maven-plugin` (validate) runs before `maven-checkstyle-plugin`(compile). Drawback is that repo-wide checkstyle would run at a later lifecycle phase.
- Perhaps create separate “generate” profile in `spring-cloud-gcp/pom` and exclude checkstyle. So  when building with aggregator, use this profile so only `fmt-maven-plugin` is triggered.

What are your thoughts on this?

Reply:
> On your proposal for adjusting the running phase, would it work if we configure the phase of fmt-maven-plugin so that it runs during validate?

checkstyle is also on validate, so it won't run before.

>  invoking mvnw fmt:format at the end of the generate-all.sh script is also an option.

Inspired by this, I think I have a better understanding of the issue now. `fmt-maven-plugin` was not correctly configured for each of the generated modules because it's parent are set to starter and `fmt-maven-plugin` configs in aggregator does not pass down. 
I've added the plugin to `spring-cloud-gcp/pom` with default set to skip, so it won't affect build/release for the project. Then each starter module will inherit and we are able to trigger the autoformat with `mvnw com.coveo:fmt-maven-plugin:format -Dfmt.skip=false`

Reply:
Gotcha, I see the issue with both plugins being bound to validate now. This approach to suppress and trigger later in generated modules LGTM!